conftest.py

```diff
@@ -0,0 +1,14 @@
+from sybil import Sybil
+from sybil.parsers.doctest import DocTestParser
+
+readme_tester = Sybil(
+    parsers=[DocTestParser()],
+    pattern="README.md",
+)
+
+python_file_tester = Sybil(
+    parsers=[DocTestParser()],
+    pattern="src/**/*.py",
+)
+
+pytest_collect_file = (readme_tester + python_file_tester).pytest()
```

Comment:
```suggestion
pytest_collect_file = (readme_tester + python_file_tester).pytest()

```

Reply:
We should add a hook for end-of-file whitespace fixing...

Reply:
should be covered by `ruff format`

Reply:
I didn't have lefthook installed as a git hook.